test/test/notify_awaiter.cpp

```diff
@@ -5,7 +5,11 @@ using namespace Windows::Foundation;
 
 namespace
 {
+#ifdef __cpp_lib_coroutine
+    using std::suspend_never;
+#else
     using std::experimental::suspend_never;
```

Comment:
Nit: needs indentation

Reply:
What is the expected indentation here? I don't usually see ifdef blocks indented in situations like this.

Reply:
Oh wait no this is fine, it's just that GitHub mobile didn't render it correctly.